pkg/action/action_test.go

```diff
@@ -106,3 +106,38 @@ func withFallbackAnnotations() chartOption {
 		opts.Chart.Metadata.Annotations["catalog.cattle.io/release-name"] = "fleet"
 	}
 }
+
+func withMalformedSharedDeps() chartOption {
+	return func(opts *chartOptions) {
+		if opts.Chart.Metadata.Annotations == nil {
+			opts.Chart.Metadata.Annotations = make(map[string]string)
+		}
+		// this ought to have wrong indentation:
+		opts.Chart.Metadata.Annotations["hypper.cattle.io/shared-dependencies"] = `- name: vanilla-helm
+   version: "0.1.0"
+ repository: "file://testdata/vanilla-helm"`
```

Comment:
tabs vs. spaces issue?

Reply:
We need this yaml to be malformed for tests. I used the backtick for a raw string literal it to be more readable, while being obvious that it's a weird indentation.

Reply:
:+1: